src/top.rs

```diff
@@ -28,16 +28,38 @@ pub(crate) struct Topology {
 /// which orders the addrs, such that this type uniquely identifies the link
 /// between two hosts on the network.
```

Comment:
Do you want to add a bit on the sort order with loopback now in play?

Reply:
Will do.